modules/simple-payments/paypal-express-checkout.js

```diff
@@ -9,6 +9,7 @@
 /* jshint unused:false */
 var PaypalExpressCheckout = {
 	sandbox: true,
+	$purchaseMessageContainer: null,
```

Comment:
I'm not sure if that is a standard for Jetpack but I've use the `$` prefix when assigning a variable to a dom element